front-end/src/Components/Dashboard/Event.js

```diff
@@ -1,20 +1,54 @@
 import React from "react";
 import { Link } from "react-router-dom";
 import CapitalizeEvent from "../CapitalizeEvent";
+import axios from "axios";
+import { apiURL } from "../../util/apiURL";
+import { useState, useEffect } from "react";
+
+const API = apiURL();
+
+function Event({ history, event, user_id }) {
+  const [events, setEvents] = useState([]);
+
+  useEffect(() => {
+    axios
+      .get(`${API}/events/${user_id}/${event.event_id}`)
+      .then(
+        (res) => {
+          console.log(res.data);
+          setEvents(res.data.payload.event_id);
+        },
+        (error) => {
+          history.push("/not-found");
+        }
+      )
+      .catch((c) => console.warn("catch", c));
+  }, [history, event.event_id]);
+
+  const handleDelete = () => {
+    axios
+      .delete(`${API}/events/${user_id}/${event.event_id}`)
+      .then(
+        () => history.push("/dashboard"),
+        (error) => console.warn(error)
```

Comment:
Great work! One thing I notice is that you don't need to push to the dashboard route because you are already there.

Reply:
Also, a reminder that all checklist items associated with this event also need to be deleted from /checklist